src/gui/Unix/QMPlay2.appdata.xml.in

```diff
@@ -12,25 +12,28 @@
     </description>
     <url type="homepage">https://github.com/zaps166/QMPlay2</url>
     <url type="bugtracker">https://github.com/zaps166/QMPlay2/issues</url>
-    <categories>
-        <category>AudioVideo</category>
-        <category>Player</category>
-        <category>Audio</category>
-        <category>Video</category>
-        <category>Qt</category>
-    </categories>
-    <keywords>
-        <keyword>player</keyword>
-        <keyword>audio</keyword>
-        <keyword>video</keyword>
-    </keywords>
+    <url type="vcs-browser">https://github.com/zaps166/QMPlay2</url>
+    <url type="translate">https://github.com/zaps166/QMPlay2/tree/master/lang</url>
     <content_rating type="oars-1.1"/>
-    <releases>
-        <release version="@QMPLAY2_VERSION@" date="@QMPLAY2_DATE@"/>
-    </releases>
     <project_license>LGPL-3.0</project_license>
     <developer_name>Błażej Szczygieł</developer_name>
+    <launchable type="desktop-id">io.github.zaps166.QMPlay2.desktop</launchable>
     <screenshots>
-        <screenshot type="default">https://raw.githubusercontent.com/zaps166/GitHubCommonContents/master/Screenshots/QMPlay2.png</screenshot>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/zaps166/GitHubCommonContents/master/Screenshots/QMPlay2.png</image>
+        </screenshot>
     </screenshots>
+    <releases>
+        <release version="@QMPLAY2_VERSION@" date="@QMPLAY2_DATE@"/>
+        <release version="24.04.07" date="2024-04-07" />
```

Comment:
If we have the date already, do we need the history, too?

Reply:
Up to developer.

Reply:
Ah, so one less place to modify before release without them :smile: